GTMAppAuth/Sources/AuthSessionDelegate.swift

```diff
@@ -30,15 +30,16 @@ public protocol AuthSessionDelegate {
 
   /// A method notifying the delegate that the authorization request failed.
   ///
-  /// Use this method to examine the error behind the failed authorization request and supply a more
-  /// custom error specifying whatever context is needed.
+  /// Use this method to examine the error behind the failed authorization request and supply a
+  /// customized error created asynchronously that specifies whatever context is needed.
   ///
   /// - Parameters:
   ///   - authSession: The `AuthSession` whose authorization request failed.
   ///   - originalError: The original `Error` associated with the failure.
-  /// - Returns: The new error for `AuthSession` to send back or nil if no error should be sent.
+  ///   - completion: An escaping closure to pass back the updated error.
   @objc optional func updatedError(
```

Comment:
Now that the method does not return an error synchronously, it should probably be named `updateError`. You may make `originalError` the first parameter too to make this method reads more smoothly.

Reply:
Changed to `updateError`, but have kept `authSession as the first parameter since I think we typically have the delegating object as the first argument in delegate methods like this.